libs/utils/platforms/pixel_energy.py

```diff
@@ -0,0 +1,187 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright (C) 2016, ARM Limited and contributors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from energy_model import ActiveState, EnergyModelNode, PowerDomain, EnergyModel
+
+from collections import OrderedDict
+
+silver_cpu_active_states = OrderedDict([
+    (307200,      ActiveState(capacity=149, power=90)),
+    (384000,      ActiveState(capacity=188, power=111)),
+    (460800,      ActiveState(capacity=225, power=133)),
+    (537600,      ActiveState(capacity=257, power=160)),
+    (614400,      ActiveState(capacity=281, power=182)),
+    (691200,      ActiveState(capacity=315, power=210)),
+    (768000,      ActiveState(capacity=368, power=251)),
+    (844800,      ActiveState(capacity=406, power=306)),
+    (902400,      ActiveState(capacity=428, power=332)),
+    (979200,      ActiveState(capacity=469, power=379)),
+    (1056000,     ActiveState(capacity=502, power=438)),
+    (1132800,     ActiveState(capacity=538, power=494)),
+    (1209600,     ActiveState(capacity=581, power=550)),
+    (1286400,     ActiveState(capacity=611, power=613)),
+    (1363200,     ActiveState(capacity=648, power=670)),
+    (1440000,     ActiveState(capacity=684, power=752)),
+    (1516800,     ActiveState(capacity=729, power=848)),
+    (1593600,     ActiveState(capacity=763, power=925)),
+])
+
+silver_cluster_active_states = OrderedDict([
+    (307200, ActiveState(power=4)),
+    (384000, ActiveState(power=4)),
+    (460800, ActiveState(power=4)),
+    (537600, ActiveState(power=4)),
+    (614400, ActiveState(power=4)),
+    (691200, ActiveState(power=4)),
+    (768000, ActiveState(power=8)),
+    (844800, ActiveState(power=9)),
+    (902400, ActiveState(power=15)),
+    (979200, ActiveState(power=16)),
+    (1056000, ActiveState(power=21)),
+    (1132800, ActiveState(power=22)),
+    (1209600, ActiveState(power=29)),
+    (1286400, ActiveState(power=32)),
+    (1363200, ActiveState(power=42)),
+    (1440000, ActiveState(power=49)),
+    (1516800, ActiveState(power=41)),
+    (1593600, ActiveState(power=52)),
+])
+
+gold_cpu_active_states = OrderedDict([
+    (307200,     ActiveState(capacity=149, power=93)),
```

Comment:
Here it is... we have same lower OPP capacities but different max OPP capacities for SILVER and GOLD.
Isn't the EnergyModel::littlest_cpus() returning all the CPUs while  EnergyModel::biggest_cpus() returns only the GOLD ones? Does this affects workloads generated by some tests?